### PR TITLE
LEF-610: list collection assets

### PR DESCRIPTION
**Pull request by msalvatti**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @friggframework/api-module-frontify@0.0.16-canary.210.87899d7.0
  # or 
  yarn add @friggframework/api-module-frontify@0.0.16-canary.210.87899d7.0
  ```
</details>

### CODE REVIEW

File: api-module-library/frontify/api.js [hunk 1]
```diff
@@ -343,6 +343,41 @@ class Api extends OAuth2Requester {
...
+                          assets	{
+                              items	{
+                                id
+                              title
+                                description
+                              tags {
+                                source
+                                value
+                              }
+                              __typename
```
Comment (roboli) [inline api-module-library/frontify/api.js:365]: Perhaps we need to render all properties here too, using `_filesQuery` method? cc @leofmds

Comment (msalvatti) [reply api-module-library/frontify/api.js:365]: Ok @roboli, I included ${this._filesQuery()} according to the other queries that contain assets. Thanks.

**Review by roboli — APPROVED** [review]
Looks good :+1:

**Review by leofmds — APPROVED** [review]
👍🏻

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-frontify@0.0.16`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-frontify%400.0.16) :rocket:

**Pull request merged**